proposals/kubeflow-technical-oversight-committee.md

```diff
@@ -0,0 +1,184 @@
+# Kubeflow Technical Oversight Committee
+
+The Kubeflow Technical Oversight Committee (TODO: TOC or KTOC ?) is responsible for
+cross-cutting product and design decisions.
+
+The governance of Kubeflow is an open, living document, and will continue to evolve as the
+community and project change.
+
+## Charter
+
+- Project direction and roadmap:
+
+  - Set the overall technical direction and Kubeflow roadmap.
+
+  - Create proposals based on TOC discussions and bring them to the relevant working groups for discussion.
+
+  - Working with working groups on Kubeflow cross-component integrations.
+
+  - Resolving issues that, despite best efforts, a working group has been unable to resolve.
+
+- Project releases, security, and stability:
+
+  - Ensure the stability of Kubeflow releases.
+
```

Comment:
I like this goal but I am not sure how the TOC fits in.   Most issues will be at the WG level and theirs to resolve.   Will the TOC be responsible to resolve or just additional contributing resources under direction of the WG

Reply:
I think, stability covers all of this @StefanoFioravanzo. Also, we can say about Kubeflow LTS. Should we explicitly add it to charter ?

Reply:
Oh you bring up a good point! We could add another bullet along the lines of "- Define a long term support (LTS) community SLA and support WGs for security escalations and critical bug fixes"